tests/test_benchmarks_badge.py

```
@@ -90,13 +90,22 @@ def test_skipped_counts_as_not_ok(tmp_path):
     assert text.startswith("reproducibility: ⚠️")
 
 
-def test_canonical_v200_partial(tmp_path):
-    """Sanity: today's checked-in canonical reports 6/11."""
+def test_canonical_v200_full_pass(tmp_path):
+    """Sanity: today's checked-in canonical reports 11/11 ok.
+
+    Was 6/11 during the partial first-pass calibration on 2026-05-07
+    (LoCoMo data missing + StructMemEval × 4 hitting the #473
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)